### PR TITLE
Remove Nats

### PR DESCRIPTION
**Pull request by d80tb7**
Completely Removes the NATS dependency  from Armada:

* Removes all NATS processing from server
* Removes the old "Legacy Event" Api that was populated from NATS
* Removes NATS processing from Lookout.
* Removes instantiation of Nats in E2E tests and LocalDev.

### CODE REVIEW

File: internal/armada/processor/event_nats.go [hunk 1]
```diff
@@ -2,24 +2,20 @@ package processor
...
 
 type StreamEventStore struct {
-	stream             eventstream.EventStream
 	PulsarSubmitServer *server.PulsarSubmitServer
 }
 
-func NewEventStore(stream eventstream.EventStream) *StreamEventStore {
-	return &StreamEventStore{stream: stream}
+func NewEventStore() *StreamEventStore {
+	return &StreamEventStore{}
```
Comment (JamesMurkin) [inline internal/armada/processor/event_nats.go:18]: This file should probably be renamed

Also in ReportEvents - we should assume pulsar is enabled

Comment (d80tb7) [reply internal/armada/processor/event_nats.go]: done

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "3.8"
+xversion: "3.8"
```
Comment (JamesMurkin) [inline docker-compose.yaml:1]: Is this an intentional change?

Comment (d80tb7) [reply docker-compose.yaml]: no- a typo

File: config/armada/config.yaml [hunk 3]
```diff
@@ -20,19 +20,12 @@ redis:
...
-  password: ""
-  db: 0
-  poolSize: 1000
 eventsApiRedis:
   addrs:
     - redis:6379
   password: ""
   db: 1
   poolSize: 1000
-forceNewEvents: true
```
Comment (severinson) [inline config/armada/config.yaml:35]: Do we no longer need this flag?

File: makefile [hunk 4]
```diff
@@ -486,7 +485,7 @@ tests-e2e: build-armadactl build-docker-no-lookout tests-e2e-setup
 		docker logs executor
 		echo -e "\nserver logs:"
 		docker logs server
-		docker rm -f nats redis pulsar server executor postgres lookout-ingester-migrate lookout-ingester jobservice
+		docker rm -f redis pulsar server executor postgres lookout-ingester-migrate lookout-ingester jobservice
```
Comment (severinson) [inline makefile:488]: event-ingester missing from this.

**Review by severinson — APPROVED** [review]

File: config/armada/config.yaml (hunk 3 above)
Comment (d80tb7) [reply config/armada/config.yaml:35]: we do not- the old event store was populated from nats so will now be state- you *have* to use the new events

File: e2e/setup/event-ingester-config.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+redis:
```
[the remark this answers is not in the document]
Comment (d80tb7) [reply e2e/setup/event-ingester-config.yaml:1]: ok let me remove this as one of my next PRs

File: makefile (hunk 4 above)
Comment (d80tb7) [reply makefile:488]: ok I'll fix this as part of a new PR

**Pull request merged**